pkgs/by-name/ir/ironbar/package.nix

```
@@ -17,20 +17,26 @@
 , rustPlatform
 , lib
 , fetchFromGitHub
+, luajit
+, luajitPackages
+, libpulseaudio
+, features ? []
 }:
 
+let hasFeature = f: features == [ ] || builtins.elem f features;
+in
```

```suggestion
let
  hasFeature = f: features == [ ] || builtins.elem f features;
in
```
`let` should be on its own line. Remember to use `[ ]` instead of `[]` everywhere else as well